app/Http/Controllers/WebhookController.php

```diff
@@ -116,27 +128,39 @@ public function destroy(Webhook $webhook)
         $this->authorize('delete', $webhook);
         if ($webhook->payloads->count() > 0) {
             return redirect()->back()
-                ->with('messageFail', 'This webhook has some payloads to be related with, please delete them first');
+                ->with('messageFail', [
+                    'status' => 'Delete failed',
+                    'message' => 'This webhook has some payloads to be related with, please delete them first'
+                ]);
         }
 
         if ($webhook->mappings->count() > 0) {
             return redirect()->back()
-                ->with('messageFail', 'This webhook has some mappings to be related with, please delete them first');
+                ->with('messageFail', [
+                    'status' => 'Delete failed',
+                    'message' => 'This webhook has some mappings to be related with, please delete them first'
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.NoComma</br>**Severity**: error</br>**File**: app/Http/Controllers/WebhookController.php</br>**Line**: 141</br>Each line in an array declaration must end in a comma